script/function.js

```diff
@@ -0,0 +1,36 @@
+import userData from "./emailData.js";
+
+function createTag(target, message) {
+  //태그 생성
+  if (target.nextSibling) {
+    return;
+  }
+  target.classList = "redBorder";
+  const span = document.createElement("span");
+  span.innerHTML = `${message}`;
+  span.classList.add("redMessage");
+  target.parentElement.append(span);
+}
+function removeTag(target) {
+  //태그 삭제
+  target.classList.toggle("redBorder", 0);
+  if (target.nextSibling) {
+    target.nextSibling.remove();
+  }
+}
+function checkEmail(str) {
+  //이메일 정규식
+  const reg_email =
+    /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
+  if (!reg_email.test(str)) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
이런 경우에는 아래처럼 축약해서 많이 작성하니 참고하시면 좋겠네요!
```suggestion
  return reg_email.test(str)
```